guest-agents/vars.yaml

```diff
@@ -6,3 +6,5 @@ GLIB_VERSION: 2.79.2
 PCRE2_VERSION: 10.42
 # renovate: datasource=github-releases extractVersion=^v(?<version>.*)$ depName=xenserver/xe-guest-utilities
 XE_GUEST_UTILITIES_VERSION: 8.4.0
```

Comment:
should we remove the old one?

Reply:
I think so, will reduce the confusion for new users